dhcpv4/ztpv4/ztp_test.go

```diff
@@ -4,10 +4,11 @@ import (
 	"testing"
 
 	"github.com/insomniacslk/dhcp/dhcpv4"
+	"github.com/insomniacslk/dhcp/iana"
 	"github.com/stretchr/testify/require"
 )
 
-func TestParseV4VendorClass(t *testing.T) {
+func TestParseV4ClassIdentifier(t *testing.T) {
```

Comment:
Since you are changing the name, would you mind removing the V4? I think it is redindant.